voidcrew/_DEFINES/ship_defines.dm

```diff
@@ -1,4 +1,5 @@
 #define NEUTRAL_SHIP "NEU"
+#define HOSTILE_SHIP "KOS"
```

Comment:
We actually don't want to add KOS ships, we expect NEU to fill what KOS did previously.